pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -200,8 +200,10 @@ func buildMixerInboundTCPFilter(env *model.Environment, node *model.Proxy, insta
 
 // defined in install/kubernetes/helm/istio/charts/mixer/templates/service.yaml
 const (
-	mixerPortName     = "grpc-mixer"
-	mixerMTLSPortName = "grpc-mixer-mtls"
+	//mixerPortName       = "grpc-mixer"
```

Comment:
Maybe make it a var and allow some env override ? Not sure if port numbers will be stable enough (in particular with multi-cluster).
Technically it's the service port - we could map it in EDS.